adapter/internal/discovery/xds/rate_limiter_cache.go

```diff
@@ -247,16 +250,26 @@ func (r *rateLimitPolicyCache) generateRateLimitConfig(label string) *rls_config
 	}
 
 	//Iterate through the subscription policies and append it to the orgDescriptors
-	for metadataType, metadataPolicyMap := range r.metadataBasedPolicies {
-		var subscriptionDescriptors []*rls_config.RateLimitDescriptor
-		for _, subscriptionPolicyDescriptor := range metadataPolicyMap {
-			subscriptionDescriptors = append(subscriptionDescriptors, subscriptionPolicyDescriptor)
-		}
-		metadataDescriptor := &rls_config.RateLimitDescriptor{
-			Key:         metadataType,
-			Descriptors: subscriptionDescriptors,
+	//organization -> subscription -> policyName -> RateLimitDescriptor
```

Comment:
Shall we update this comment with a simple YAML definition of a Rate-Limiter? It is more descriptive than this comment.